src/components/timeline/TrackMemory.js

```diff
@@ -48,72 +49,27 @@ type DispatchProps = {||};
 
 type Props = ConnectedProps<OwnProps, StateProps, DispatchProps>;
 
-type State = {
+type State = {|
   hoveredCounter: null | number,
   mouseX: CssPixels,
   mouseY: CssPixels,
-};
+|};
```

Comment:
nit: can you please move these Props and State definition closer to the class they're used for?

Reply:
Yeah, I also documented the Canvas component.